modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc

```diff
@@ -0,0 +1,24 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/getting-started-with-openshift-gitops.adoc
+
+[id="logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials_{context}"]
+= Logging in to the Argo CD instance by using your OpenShift credentials
+
+Red Hat OpenShift GitOps Operator automatically creates a ready-to-use Argo CD instance that is available in the `openshift-gitops` namespace.
+
+.Prerequisites
+
+* You have installed the Red Hat OpenShift GitOps Operator in your cluster.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to *Operators* -> *Installed Operators*.
+. Navigate to *Red Hat applications* -> *Application Stages* -> *Argo CD*. The login page of the Argo CD UI is displayed in a new window.
```

Comment:
@Preeticp I can specify that it's located near the bell icon in the upper right corner. This might make it a little clearer. WDYT?

Reply:
Hi Yana, the 'Click (screenshot of icon) icon' part of my suggestion did not show the content in the parenthesis as i used angular brackets instead of the curved one. Not sure why GH does that, content within angular brackets is not shown.